lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -1057,8 +1057,11 @@ def customize_vm(self, vm_obj):
             if 'joindomain' in self.params['customization']:
                 # TODO: Escalate if domainAdmin and domainPassword are not provided
```

Comment:
While you are at it, maybe fix that TODO as well.
I have no environment to test that properly.